Gemfile

```diff
@@ -1,5 +1,5 @@
 source ENV['GEM_SOURCE'] || "https://rubygems.org"
-puppet_branch = ENV['puppet_branch'] || "latest"
+puppet_branch = ENV['PUPPET_VERSION'] || "latest"
```

Comment:
puppetlabs/ci-job-configs#469 adds new pipelines that includes spec tests which use this new variable name.